docs/how-to-guides/environment-setup-and-maintenance/adding-usgscsm-plugin-to-isis.md

```diff
@@ -0,0 +1,78 @@
+# Adding USGSCSM Plugin to ISIS
+
+## For users
+You can install the latest [USGSCSM](https://github.com/DOI-USGS/usgscsm) library via conda:
+
+```sh
+conda install conda-forge::usgscsm
+```
```

Comment:
Does one need to do this if they install the LTS? How about the most recent version? We should note if and when one needs to do this as an independent step.

Reply:
As of 8.0 LTS and 8.1 feature release yes, going forward we will add USGSCSM to the recipe/meta.yaml so that USGSCSM comes with ISIS.

It is worth noting that here that all versions pre 8.0 and 8.1 will need to do this manually

Reply:
I think noting that in these docs would be helpful to many users? We know how fast different groups update (myself included!)....

Reply:
@chkim-usgs Apologies if this is pedantic, but should this read something like `ISIS version 8.0LTS, 8.1, and earlier _require_...`. Just to cover the random person running 7.1 somewhere and getting confused because their version doesn't start with 8.